### PR TITLE
bpo-34449: HP aCC complains about invalid -fPIC on HP-UX

### PR DESCRIPTION
**Pull request by michael-o**
-fPIC is code generation option and not a link option. At compile time '+z' is
already properly used with HP aCC and shared libraries are correctly linked
with '+b'. This switch can safely be dropped.

Patch by Michael Osipov.

https://bugs.python.org/issue34449

### CODE REVIEW

**Comment by erlend-aasland** [comment]
@michael-o, can you please sign the CLA?

**Comment by michael-o** [comment]
@erlend-aasland I am certain that I already did because other PRs from me have been absorbed into Python already. See here: https://github.com/python/cpython/commits?author=michael-o

**Comment by michael-o** [comment]
Ah, it was a different email address. This I can change.

**Comment by erlend-aasland** [comment]
FYI, CLA is still failing. We cannot merge without it.

**Comment by michael-o** [comment]
@erlend-aasland Just signed. Please check.

**Comment by erlend-aasland** [comment]
@tiran, what do you think of this? IMO, it is nice to slim down setup.py, even though its just a couple of lines. OTOH, most of this code will probably go soon anyway (gh-90005). But, having less stuff to think about for gh-90005 is also a good thing. So I'm +1 :)

**Review by tiran — APPROVED** [review]
SGTM

File: Misc/NEWS.d/next/Build/2018-08-21-11-10-18.bpo-34449.Z3qm3c.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Drop invalid compiler switch for HP aCC on HP-UX. Patch by Michael Osipov.
```
Comment (erlend-aasland) [inline Misc/NEWS.d/next/Build/2018-08-21-11-10-18.bpo-34449.Z3qm3c.rst]:
```suggestion
Drop invalid compiler switch ``-fPIC`` for HP aCC on HP-UX. Patch by Michael Osipov.
```

**Review by erlend-aasland — APPROVED** [review]

**Pull request merged**